core/vm/hooks.libevm.go

```
@@ -74,3 +75,10 @@ func (evm *EVM) overrideEVMResetArgs(txCtx TxContext, statedb StateDB) (TxContex
 	args := libevmHooks.OverrideEVMResetArgs(evm.chainRules, &EVMResetArgs{txCtx, statedb})
 	return args.TxContext, args.StateDB
 }
+
+func overrideJumpTable(r params.Rules, jt *JumpTable) *JumpTable {
+	if libevmHooks == nil {
+		return jt
+	}
```

I'm thinking, if a user calls `overrideJumpTable`, maybe we should panic if the libevmHooks is nil? It sounds a bit strange to pass in rules which just gets silently discarded by this function if the hooks are nil 🤔 We could handle the libevmHooks nil check at the calling layer instead?